### ISSUE TITLE
Complete translation - files below 50% translation need attention

### ISSUE THREAD

**Issue by arcturusannamalai** [Ezhil-Language-Foundation/RubyKin#15]
stats.py - show only MD files
0.0000% => /Users/muthu/devel/RubyKin/enumerables/each.md
0.0000% => /Users/muthu/devel/RubyKin/enumerables/map.md
0.0000% => /Users/muthu/devel/RubyKin/enumerables/select.md
0.0000% => /Users/muthu/devel/RubyKin/enumerables/enumerable.md
0.0000% => /Users/muthu/devel/RubyKin/enumerables/each_with_index.md
0.8242% => /Users/muthu/devel/RubyKin/collections/hash.md
1.5098% => /Users/muthu/devel/RubyKin/answers_to_practice_problems/answers.md
17.6471% => /Users/muthu/devel/RubyKin/appendix/contact.md
18.4765% => /Users/muthu/devel/RubyKin/numbers/number.md
45.5882% => /Users/muthu/devel/RubyKin/strings/practice.md

**Comment by y4yuvaraj** [comment]
Hi any help needed for tamil translation?

**Comment by arcturusannamalai** [comment]
Yes @y4yuvaraj - please submit a pull request with your translations
